CONTRIBUTING.md

```diff
@@ -51,8 +51,24 @@ The following table explains how we interpret each of the states for a Pivotal s
 
 ### Commits
 
-- Follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
-  - (e.g) Prefix commits with `fix:`, `feat:`, `docs:` `chore:`
+Commits should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+
+The commit types should conform to:
```

Comment:
I'm adding a suggestion that shows the list below contains examples and isn't exhaustive, as it's missing additional types we use regularly like docs, ci, style, etc.

```suggestion
Commits should conform to the [Conventional Commits spec](https://www.conventionalcommits.org/en/v1.0.0/), with examples shown below. See the spec for a full list of types.```